package.json

```diff
@@ -1,8 +1,8 @@
 {
-  "name": "create-radix-app",
-  "version": "0.2.0",
+  "name": "@radixdlt/create-radix-app",
```

Comment:
The package name should be **create-radix-app**. There should be no **@radixdlt/**